src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -399,6 +408,52 @@ export const VolunteerFormCalendar = ({
   );
 };
 
+const fileFilters = (
+  fileRegex: any,
+  calendarData: {
+    id: number;
+    content: { files: any[] };
+  }
+) =>
+  JSON.stringify(
+    calendarData?.content?.files
+      .map(({ file_name: infoText, mime_type: mimeType, url: uri, id: fileId }: any) => {
+        const isFile = fileRegex.test(infoText);
+
+        if (isFile) return { entryId: calendarData?.id, fileId, infoText, mimeType, uri };
+        else return;
+      })
+      .filter((otherFiles: any) => otherFiles != null)
+  );
+
+const filerParseAndUpload = (calendarNewData: VolunteerCalendar, id: number) => {
+  const images = jsonParser(calendarNewData.images);
+  const documents = jsonParser(calendarNewData.documents);
+  const uris: { uri: StringConstructor; mimeType: StringConstructor }[] = [];
+  if (images?.length) {
+    images.forEach(({ uri = String, mimeType = String }) => {
```

Comment:
what was the aim with `= String` here? this sets a default value, right? i guess you wanted to set a type here?

```
images.forEach(({ uri, mimeType }: { uri: string; mimeType: string }) => {
  uris.push({ uri, mimeType });
});
```

Reply:
01b627a

Reply:
Yes, I wanted to specify the type, but I didn't know how to do it. Thank you.